tools/pythonpkg/setup.py

```diff
@@ -154,6 +154,12 @@ def __str__(self):
         import numpy
         return numpy.get_include()
 
+#TODO: add missing third party includes
+# Add paths that should be included when building the python client here
+def extra_python_includes():
+	extra_includes = []
+	extra_includes += [os.path.join(script_path, '..', '..', 'third_party', 'fmt', 'include')] #third_party/fmt/include
```

Comment:
Ah good idea, I had found this list in there, but wasn't sure of how to get it cleanly into both

Reply:
Update: setup.py is not a fan (as in: it won't let you) of bundling files outside of the root directory, even through MANIFEST.in

So I would have to move package_build.py to tools/pythonpkg and also python_helpers.py because package_build depends on that.
Which would result in having to rewrite a bunch of code in amalgamation.py, setup.py and package_build + possibly others, because they rely on these files being in certain locations for relative paths. Because of this I think this should be its own PR.

Reply:
Agreed, let's revert to the original

Reply:
Hmm after looking at it more closely, it's only needed in a less broad scope as where I was including it, it seems to work now ?